packages/gensx-core/src/workflow-state.ts

```diff
@@ -38,13 +38,13 @@ export interface WorkflowDataMessage {
 
 export interface WorkflowEventMessage {
   type: "event";
-  data: Record<string, JsonValue>;
+  data: JsonValue;
   label: string;
 }
 
 export interface WorkflowObjectMessage {
   type: "object";
-  data: Record<string, JsonValue>;
+  data: JsonValue;
```

Comment:
Are we ok with a valid value being `string` or `number`, etc?

Reply:
I think that's fine. I could see a scenario where the object you're streaming is the status of the workflow and it's just a string or things like that. Really any type you could use with `useState` feels reasonable.

Reply:
Sounds good. These are written in at least three places, ill look at making a types package.